ajax/libs/gridly/package.json

```diff
@@ -0,0 +1,132 @@
+{
+  "name": "gridly",
+  "version": "1.1.0",
+  "description": "The minimal (~157 bytes) grid system for modern browsers.",
+  "main": "lib/gridly.css",
+  "directories": {
+    "example": "example"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/IonicaBizau/gridly.git"
+  },
+  "keywords": [
+    "gridly",
+    "grid",
+    "css",
+    "modern",
+    "browsers"
+  ],
+  "author": "Ionică Bizău <bizauionica@gmail.com> (http://ionicabizau.net)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/IonicaBizau/gridly/issues"
+  },
+  "homepage": "https://github.com/IonicaBizau/gridly#readme",
+  "blah": {
```

Comment:
Haha, hoped you will not notice that! I guess this should be part of the unit testing, shouldn't it? :joy: